components/esp_modem/examples/pppos_client/main/pppos_client_main.c

```diff
@@ -157,9 +175,25 @@ void app_main(void)
 #else
     ESP_LOGI(TAG, "Initializing esp_modem for a generic module...");
     esp_modem_dce_t *dce = esp_modem_new(&dte_config, &dce_config, esp_netif);
+#endif
+
+#elif defined(CONFIG_EXAMPLE_SERIAL_CONFIG_USB)
+    while (1) {
```

Comment:
Could we make the while loop "bigger", so it's better readable for both UART and USB?

Reply:
I agree, this change does not improve readability :(

I plan to follow up this PR for support of multiple types of DCE. Currently the USB examples support only BG96. So the USB initialization sequence could then by similar to UART. But again, would you agree with follow-up PR?